### PR TITLE
#1045 Tables on mobile WIP

### PR DESCRIPTION
**Pull request by jakeatdocforce**
One option is to stack on small devices. Hide current table headers and add headers to rows.
Will need to add divs above tables to id each table for naming.  Only 2 tables in this as a test.
Other tables will hide headers completely and full width content.

### CODE REVIEW

**Comment by mDuo13** [comment]
Tried building it and I'm not seeing much difference on mobile in the "Special Accounts" table. The table cells have a little bit more space but aren't getting stacked.

**Comment by jakeatdocforce** [comment]
Should look like this. 

Screen Shot 2021-05-10 at 4 15 57 PM

**Comment by mDuo13** [comment]
First of all, the overall approach looks a little fragile. Having to add a unique div and some JS code for each wide table is not optimal, though I imagine some of that could probably be made more generic with some JS and maybe a Dactyl filter.

Second, the result still isn't working quite right, as you can see "Black Hole" and "Yes" are overlapping in this image:



Actually, I did some searching and found out that [Bootstrap's `table-responsive` class](https://getbootstrap.com/docs/4.0/content/tables/#responsive-tables) makes the table horizontally scrollable, which is probably what we want for most or all tables in the docs. So we can probably either copy the style rules for that for `.content table` or we can find a way to add that class to tables in the content, like with a Dactyl filter.

**Comment by jakeatdocforce** [comment]
correct I don't think its a useable or scaleable strategy.  Was thinking of a separate way using JS to make more generic for all tables. 
But if scrolling is acceptable that would be fastest/easiest to implement.

File: tool/filter_css_tables.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+################################################
+## Add class to tables
+## Author: Rome Reginelli
```
Comment (mDuo13) [inline tool/filter_css_tables.py]: Hey, take credit for your own work here! :laughing:

**Review by mDuo13 — CHANGES_REQUESTED** [review]

File: tool/filter_css_tables.py (hunk 1 above)
Comment (jakeatdocforce) [reply tool/filter_css_tables.py]: haha done

**Review by mDuo13 — APPROVED** [review]

**Comment by mDuo13** [comment]
Rebuilt and it looks good now.

**Pull request merged**